tsconfig.json

```diff
@@ -1,5 +1,13 @@
 // This tsconfig is only used for tooling (ex: typecheck in code editors)
 {
   "extends": "./tsconfig.base.json",
-  "include": ["./packages/**/src/**/*.ts", "./packages/**/test/**/*.ts"]
+  "include": [
+    "./*.js",
+    "./packages/**/*.js",
+    "./packages/**/*.ts",
+    "./scripts/**/*.js",
+    "./test/*.js",
+    "./test/unit/*.js",
+    "./.eslintrc.js"
+  ]
```

Comment:
This too: could it be simplified or removed?

Reply:
Right, this could be simplified as
```
  "include": [".", ".eslintrc.js"],
  "exclude": ["./test/e2e"]
  ```